docs/index.md

```diff
@@ -6,10 +6,14 @@ Plugins can be used as:
 * frontend pages - a completely new page with own features and UI can be shown to authenticated UCRM users or clients. This can be used for a custom features, for example user-defined reports, batch data changes with user input, etc.
 
 ## Plugin samples for an easy start of creating your own plugin
-* [Plugin Template](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/plugin-template) - Simpliest UCRM plugin sample, can be used as a base for a new plugin development.
+* [Plugin Template](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/plugin-template) - Simplest UCRM plugin sample, can be used as a base for a new plugin development.
 * [Invoice CSV Export](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/invoice-csv-export) - Better plugin example, actually doing something: a new menu item shown, enables user to define From & To date filters, exports filtered invoices into CSV.  
 * [Revenue Report](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/revenue-report) - Revenue report grouped by products or services, shown under the Reporting main menu section.
 
+## UCRM Plugin SDK
+You can use [UCRM Plugin SDK](https://github.com/Ubiquiti-App/UCRM-Plugin-SDK) to help you with development of plugins.  
+It contains classes able to help you with calling UCRM API, getting plugin's configuration and much more.
```

Comment:
Too complicated, refactor:
"It contains classes to call UCRM API, to get plugin's configuration, and more."